tremor-pipeline/src/lib.rs

```diff
@@ -121,6 +194,26 @@ pub struct MetricsMsg {
     pub origin_uri: Option<EventOriginUri>,
 }
 
+#[derive(Debug, Clone)]
+/// Language from which the logs are coming (Rust, Tremor, etc.)
+pub enum LanguageKind {
```

Comment:
Perhaps `LogSource` would be a better name here as it has nothing really to do with languages

Reply:
also cleaned up 👍